website/src/contribute/release-guide.md

```diff
@@ -428,20 +419,59 @@ Javadoc to the Javadoc for other modules that Beam depends on.
   the version number has changed, download a new version of the corresponding
   `<module>-docs/package-list` file.
 
+### Triage release-blocking issues in JIRA
+
+There could be outstanding release-blocking issues, which should be triaged before proceeding to build a release candidate. We track them by assigning a specific `Fix version` field even before the issue resolved.
+
+The list of release-blocking issues is available at the [version status page](https://issues.apache.org/jira/browse/BEAM/?selectedTab=com.atlassian.jira.jira-projects-plugin:versions-panel). Triage each unresolved issue with one of the following resolutions:
+
+For all JIRA issues:
+
+* If the issue has been resolved and JIRA was not updated, resolve it accordingly.
+
+For all `Test` JIRA issues:
```

Comment:
Can this be a look over the `flaky` label?

Reply:
Sure, I added looking over the flaky labeled tests as well. I want to keep the "Bug" JIRAs because these include the ones created from the verify_release_build step and we want a dependency on closing those out.